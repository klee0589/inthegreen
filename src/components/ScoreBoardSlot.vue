<template>
  <div class="scoreboard">
    <div class="scoreboard__container">
      <div class="scoreboard__basics mb-05">
        <span class="label">
          {{ game.competitions[0].status.type.description }}
        </span>
        <span class="text-danger">
          {{ game.competitions[0].status.displayClock }}
        </span>
      </div>
      <div class="scoreboard__teams">
        <div class="scoreboard__team scoreboard__team--align-right mr-2">
          <div class="scoreboard__badge ml-1"></div>
          <span class="scoreboard__name">{{
            game.competitions[0].competitors[0].team.displayName
          }}</span>
        </div>
        <div class="scoreboard__result">
          <span class="scoreboard__result-home">{{
            game.competitions[0].competitors[0].score
          }}</span>
          <span class="scoreboard__result-separator">:</span>
          <span class="scoreboard__result-home">{{
            game.competitions[0].competitors[1].score
          }}</span>
        </div>
        <div class="scoreboard__team scoreboard__team--align-left ml-2">
          <div class="scoreboard__badge mr-1"></div>
          <span class="scoreboard__name">{{
            game.competitions[0].competitors[1].team.displayName
          }}</span>
        </div>
      </div>
    </div>
  </div>
</template>

<script>
export default {
  props: ["game"]
};
</script>

<style lang="scss">
$gray300: #d2d2d2;
$gray500: #6e6e6e;
$gray700: #212121;
$gray900: #0f0f0f;
$blue: #29bdde;
$lightBlue: #31a49b;
$red: #ff3d3d;
$border-radius: 4px;
$lightGreen: #507450;
$green: #29502c;
$darkRed: #be2c2c;
$space-size: 8px;
$space-size-2: $space-size * 2;
$space-size-3: $space-size * 3;
$space-size-4: $space-size * 4;
$space-size-5: $space-size * 5;
$space-size-6: $space-size * 6;

$white: $gray300;
$secondaryTextColor: $gray500;
$black: $gray900;
$bodyBg: $gray700;
$borderColor: $gray700;
$primaryColor: $blue;
.text-danger {
  color: $red;
}
.label {
  font-size: 9px;
  text-transform: uppercase;
  color: $secondaryTextColor;
  font-weight: 700;
  letter-spacing: 0.095em;
  margin-bottom: $space-size / 2;
}
.mb-05 {
  margin-bottom: $space-size / 2;
}
.mb-0 {
  margin-bottom: 0 !important;
}
.mb-1 {
  margin-bottom: $space-size;
}
.mt-1 {
  margin-top: $space-size;
}
.mt-2 {
  margin-top: $space-size-2;
}
.ml-1 {
  margin-left: $space-size;
}
.ml-2 {
  margin-left: $space-size-2;
}
.ml-3 {
  margin-left: $space-size-3;
}
.ml-4 {
  margin-left: $space-size-4;
}
.ml-5 {
  margin-left: $space-size-5;
}

.mr-1 {
  margin-right: $space-size;
}
.mr-2 {
  margin-right: $space-size-2;
}
.mr-3 {
  margin-right: $space-size-3;
}
.mr-4 {
  margin-right: $space-size-4;
}
.mr-5 {
  margin-right: $space-size-5;
}
.scoreboard {
  background: $black;
  margin-bottom: $space-size;
  border-radius: $border-radius;
  &__container {
    padding: $space-size-2;
  }
  &__result {
    text-align: center;
    display: flex;
    align-items: center;
    justify-content: center;
  }
  .scoreboard__result-home,
  .scoreboard__result-away,
  .scoreboard__result-separator {
    font-size: 26px;
  }
  .scoreboard__result-separator {
    margin: 0 $space-size;
    color: $secondaryTextColor;
  }
  &__badge {
    width: 38px;
    height: 38px;
    img {
      height: 100%;
    }
  }
  &__teams {
    display: flex;
    justify-content: center;
  }
  &__team {
    display: flex;
    align-items: center;
    justify-content: flex-start;
    flex-grow: 1;
    &--align-right {
      flex-direction: row-reverse;
      .scoreboard__name {
        text-align: right;
      }
    }
    &--align-left {
      .scoreboard__name {
        text-align: left;
      }
    }
  }
  &__name {
    display: block;
    width: 100px;
  }

  &__nav {
    border-top: 1px solid $borderColor;
    display: flex;
    align-items: center;
    justify-content: space-around;
    padding: 0 $space-size-2;
    margin: 0;
  }
  &__basics {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
  }
}
</style>
