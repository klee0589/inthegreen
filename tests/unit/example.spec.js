import { mount } from "@vue/test-utils";
import ScoreBoardSlot from "@/components/ScoreBoardSlot.vue";

test("component is showing", async () => {
  const wrapper = mount(ScoreBoardSlot);
  await wrapper.setProps({
    game: {
      id: "401220335",
      uid: "s:20~l:28~e:401220335",
      date: "2020-11-24T01:15Z",
      name: "Los Angeles Rams at Tampa Bay Buccaneers",
      shortName: "LAR @ TB",
      season: { year: 2020, type: 2 },
      competitions: [
        {
          id: "401220335",
          uid: "s:20~l:28~e:401220335~c:401220335",
          date: "2020-11-24T01:15Z",
          attendance: 0,
          type: { id: "1", abbreviation: "STD" },
          timeValid: true,
          neutralSite: false,
          conferenceCompetition: false,
          recent: false,
          venue: {
            id: "3886",
            fullName: "Raymond James Stadium",
            address: { city: "Tampa", state: "FL" },
            capacity: 65618,
            indoor: false,
          },
          competitors: [
            {
              id: "27",
              uid: "s:20~l:28~t:27",
              type: "team",
              order: 0,
              homeAway: "home",
              team: {
                id: "27",
                uid: "s:20~l:28~t:27",
                location: "Tampa Bay",
                name: "Buccaneers",
                abbreviation: "TB",
                displayName: "Tampa Bay Buccaneers",
                shortDisplayName: "Buccaneers",
                color: "A80D08",
                alternateColor: "34302b",
                isActive: true,
                venue: { id: "3886" },
                links: [
                  {
                    rel: ["clubhouse", "desktop", "team"],
                    href:
                      "http://www.espn.com/nfl/team/_/name/tb/tampa-bay-buccaneers",
                    text: "Clubhouse",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["roster", "desktop", "team"],
                    href:
                      "http://www.espn.com/nfl/team/roster/_/name/tb/tampa-bay-buccaneers",
                    text: "Roster",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["stats", "desktop", "team"],
                    href:
                      "http://www.espn.com/nfl/team/stats/_/name/tb/tampa-bay-buccaneers",
                    text: "Statistics",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["schedule", "desktop", "team"],
                    href: "http://www.espn.com/nfl/team/schedule/_/name/tb",
                    text: "Schedule",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["photos", "desktop", "team"],
                    href: "http://www.espn.com/nfl/team/photos/_/name/tb",
                    text: "photos",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["scores", "sportscenter", "app", "team"],
                    href:
                      "sportscenter://x-callback-url/showClubhouse?uid=s:20~l:28~t:27&section=scores",
                    text: "Scores",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["draftpicks", "desktop", "team"],
                    href:
                      "http://www.espn.com/nfl/draft/teams/_/name/tb/tampa-bay-buccaneers",
                    text: "Draft Picks",
                    isExternal: false,
                    isPremium: true,
                  },
                  {
                    rel: ["transactions", "desktop", "team"],
                    href: "http://www.espn.com/nfl/team/transactions/_/name/tb",
                    text: "Transactions",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["injuries", "desktop", "team"],
                    href: "http://www.espn.com/nfl/team/injuries/_/name/tb",
                    text: "Injuries",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["depthchart", "desktop", "team"],
                    href: "http://www.espn.com/nfl/team/depth/_/name/tb",
                    text: "Depth Chart",
                    isExternal: false,
                    isPremium: false,
                  },
                ],
                logo:
                  "https://a.espncdn.com/i/teamlogos/nfl/500/scoreboard/tb.png",
              },
              score: "0",
              linescores: [{ value: 0 }],
              statistics: [],
              records: [
                {
                  name: "All Splits",
                  abbreviation: "Any",
                  type: "total",
                  summary: "7-3",
                },
                { name: "Home", type: "home", summary: "3-1" },
                { name: "Road", type: "road", summary: "4-2" },
              ],
              leaders: [
                {
                  name: "passingLeader",
                  displayName: "Passing Leader",
                  shortDisplayName: "PASS",
                  abbreviation: "PYDS",
                  leaders: [
                    {
                      displayValue: "254-385, 2739 YDS, 23 TD, 7 INT",
                      value: 2739,
                      athlete: {
                        id: "2330",
                        fullName: "Tom Brady",
                        displayName: "Tom Brady",
                        shortName: "T. Brady",
                        links: [
                          {
                            rel: ["playercard", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/2330/tom-brady",
                          },
                          {
                            rel: ["stats", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/stats/_/id/2330/tom-brady",
                          },
                          {
                            rel: ["splits", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/splits/_/id/2330/tom-brady",
                          },
                          {
                            rel: ["gamelog", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/gamelog/_/id/2330/tom-brady",
                          },
                          {
                            rel: ["news", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/news/_/id/2330/tom-brady",
                          },
                          {
                            rel: ["bio", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/bio/_/id/2330/tom-brady",
                          },
                          {
                            rel: ["overview", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/2330/tom-brady",
                          },
                        ],
                        headshot:
                          "https://a.espncdn.com/i/headshots/nfl/players/full/2330.png",
                        jersey: "12",
                        position: { abbreviation: "QB" },
                        team: { id: "27" },
                        active: true,
                      },
                      team: { id: "27" },
                    },
                  ],
                },
                {
                  name: "rushingLeader",
                  displayName: "Rushing Leader",
                  shortDisplayName: "RUSH",
                  abbreviation: "RYDS",
                  leaders: [
                    {
                      displayValue: "143 CAR, 730 YDS, 5 TD",
                      value: 730,
                      athlete: {
                        id: "3912550",
                        fullName: "Ronald Jones II",
                        displayName: "Ronald Jones II",
                        shortName: "R. Jones II",
                        links: [
                          {
                            rel: ["playercard", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/3912550/ronald-jones-ii",
                          },
                          {
                            rel: ["stats", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/stats/_/id/3912550/ronald-jones-ii",
                          },
                          {
                            rel: ["splits", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/splits/_/id/3912550/ronald-jones-ii",
                          },
                          {
                            rel: ["gamelog", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/gamelog/_/id/3912550/ronald-jones-ii",
                          },
                          {
                            rel: ["news", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/news/_/id/3912550/ronald-jones-ii",
                          },
                          {
                            rel: ["bio", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/bio/_/id/3912550/ronald-jones-ii",
                          },
                          {
                            rel: ["overview", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/3912550/ronald-jones-ii",
                          },
                        ],
                        headshot:
                          "https://a.espncdn.com/i/headshots/nfl/players/full/3912550.png",
                        jersey: "27",
                        position: { abbreviation: "RB" },
                        team: { id: "27" },
                        active: true,
                      },
                      team: { id: "27" },
                    },
                  ],
                },
                {
                  name: "receivingLeader",
                  displayName: "Receiving Leader",
                  shortDisplayName: "REC",
                  abbreviation: "RECYDS",
                  leaders: [
                    {
                      displayValue: "40 REC, 514 YDS, 8 TD",
                      value: 514,
                      athlete: {
                        id: "16737",
                        fullName: "Mike Evans",
                        displayName: "Mike Evans",
                        shortName: "M. Evans",
                        links: [
                          {
                            rel: ["playercard", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/16737/mike-evans",
                          },
                          {
                            rel: ["stats", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/stats/_/id/16737/mike-evans",
                          },
                          {
                            rel: ["splits", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/splits/_/id/16737/mike-evans",
                          },
                          {
                            rel: ["gamelog", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/gamelog/_/id/16737/mike-evans",
                          },
                          {
                            rel: ["news", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/news/_/id/16737/mike-evans",
                          },
                          {
                            rel: ["bio", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/bio/_/id/16737/mike-evans",
                          },
                          {
                            rel: ["overview", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/16737/mike-evans",
                          },
                        ],
                        headshot:
                          "https://a.espncdn.com/i/headshots/nfl/players/full/16737.png",
                        jersey: "13",
                        position: { abbreviation: "WR" },
                        team: { id: "27" },
                        active: true,
                      },
                      team: { id: "27" },
                    },
                  ],
                },
              ],
            },
            {
              id: "14",
              uid: "s:20~l:28~t:14",
              type: "team",
              order: 1,
              homeAway: "away",
              team: {
                id: "14",
                uid: "s:20~l:28~t:14",
                location: "Los Angeles",
                name: "Rams",
                abbreviation: "LAR",
                displayName: "Los Angeles Rams",
                shortDisplayName: "Rams",
                color: "00295B",
                alternateColor: "b3995d",
                isActive: true,
                venue: { id: "477" },
                links: [
                  {
                    rel: ["clubhouse", "desktop", "team"],
                    href:
                      "http://www.espn.com/nfl/team/_/name/lar/los-angeles-rams",
                    text: "Clubhouse",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["roster", "desktop", "team"],
                    href:
                      "http://www.espn.com/nfl/team/roster/_/name/lar/los-angeles-rams",
                    text: "Roster",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["stats", "desktop", "team"],
                    href:
                      "http://www.espn.com/nfl/team/stats/_/name/lar/los-angeles-rams",
                    text: "Statistics",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["schedule", "desktop", "team"],
                    href: "http://www.espn.com/nfl/team/schedule/_/name/lar",
                    text: "Schedule",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["photos", "desktop", "team"],
                    href: "http://www.espn.com/nfl/team/photos/_/name/lar",
                    text: "photos",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["scores", "sportscenter", "app", "team"],
                    href:
                      "sportscenter://x-callback-url/showClubhouse?uid=s:20~l:28~t:14&section=scores",
                    text: "Scores",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["draftpicks", "desktop", "team"],
                    href:
                      "http://www.espn.com/nfl/draft/teams/_/name/lar/los-angeles-rams",
                    text: "Draft Picks",
                    isExternal: false,
                    isPremium: true,
                  },
                  {
                    rel: ["transactions", "desktop", "team"],
                    href:
                      "http://www.espn.com/nfl/team/transactions/_/name/lar",
                    text: "Transactions",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["injuries", "desktop", "team"],
                    href: "http://www.espn.com/nfl/team/injuries/_/name/lar",
                    text: "Injuries",
                    isExternal: false,
                    isPremium: false,
                  },
                  {
                    rel: ["depthchart", "desktop", "team"],
                    href: "http://www.espn.com/nfl/team/depth/_/name/lar",
                    text: "Depth Chart",
                    isExternal: false,
                    isPremium: false,
                  },
                ],
                logo:
                  "https://a.espncdn.com/i/teamlogos/nfl/500/scoreboard/lar.png",
              },
              score: "0",
              linescores: [{ value: 0 }],
              statistics: [],
              records: [
                {
                  name: "All Splits",
                  abbreviation: "Any",
                  type: "total",
                  summary: "6-3",
                },
                { name: "Home", type: "home", summary: "4-0" },
                { name: "Road", type: "road", summary: "2-3" },
              ],
              leaders: [
                {
                  name: "passingLeader",
                  displayName: "Passing Leader",
                  shortDisplayName: "PASS",
                  abbreviation: "PYDS",
                  leaders: [
                    {
                      displayValue: "213-321, 2447 YDS, 13 TD, 6 INT",
                      value: 2447,
                      athlete: {
                        id: "3046779",
                        fullName: "Jared Goff",
                        displayName: "Jared Goff",
                        shortName: "J. Goff",
                        links: [
                          {
                            rel: ["playercard", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/3046779/jared-goff",
                          },
                          {
                            rel: ["stats", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/stats/_/id/3046779/jared-goff",
                          },
                          {
                            rel: ["splits", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/splits/_/id/3046779/jared-goff",
                          },
                          {
                            rel: ["gamelog", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/gamelog/_/id/3046779/jared-goff",
                          },
                          {
                            rel: ["news", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/news/_/id/3046779/jared-goff",
                          },
                          {
                            rel: ["bio", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/bio/_/id/3046779/jared-goff",
                          },
                          {
                            rel: ["overview", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/3046779/jared-goff",
                          },
                        ],
                        headshot:
                          "https://a.espncdn.com/i/headshots/nfl/players/full/3046779.png",
                        jersey: "16",
                        position: { abbreviation: "QB" },
                        team: { id: "14" },
                        active: true,
                      },
                      team: { id: "14" },
                    },
                  ],
                },
                {
                  name: "rushingLeader",
                  displayName: "Rushing Leader",
                  shortDisplayName: "RUSH",
                  abbreviation: "RYDS",
                  leaders: [
                    {
                      displayValue: "102 CAR, 486 YDS, 4 TD",
                      value: 486,
                      athlete: {
                        id: "4039359",
                        fullName: "Darrell Henderson Jr.",
                        displayName: "Darrell Henderson Jr.",
                        shortName: "D. Henderson Jr.",
                        links: [
                          {
                            rel: ["playercard", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/4039359/darrell-henderson-jr",
                          },
                          {
                            rel: ["stats", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/stats/_/id/4039359/darrell-henderson-jr",
                          },
                          {
                            rel: ["splits", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/splits/_/id/4039359/darrell-henderson-jr",
                          },
                          {
                            rel: ["gamelog", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/gamelog/_/id/4039359/darrell-henderson-jr",
                          },
                          {
                            rel: ["news", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/news/_/id/4039359/darrell-henderson-jr",
                          },
                          {
                            rel: ["bio", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/bio/_/id/4039359/darrell-henderson-jr",
                          },
                          {
                            rel: ["overview", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/4039359/darrell-henderson-jr",
                          },
                        ],
                        headshot:
                          "https://a.espncdn.com/i/headshots/nfl/players/full/4039359.png",
                        jersey: "27",
                        position: { abbreviation: "RB" },
                        team: { id: "14" },
                        active: true,
                      },
                      team: { id: "14" },
                    },
                  ],
                },
                {
                  name: "receivingLeader",
                  displayName: "Receiving Leader",
                  shortDisplayName: "REC",
                  abbreviation: "RECYDS",
                  leaders: [
                    {
                      displayValue: "53 REC, 577 YDS, 2 TD",
                      value: 577,
                      athlete: {
                        id: "2977187",
                        fullName: "Cooper Kupp",
                        displayName: "Cooper Kupp",
                        shortName: "C. Kupp",
                        links: [
                          {
                            rel: ["playercard", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/2977187/cooper-kupp",
                          },
                          {
                            rel: ["stats", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/stats/_/id/2977187/cooper-kupp",
                          },
                          {
                            rel: ["splits", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/splits/_/id/2977187/cooper-kupp",
                          },
                          {
                            rel: ["gamelog", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/gamelog/_/id/2977187/cooper-kupp",
                          },
                          {
                            rel: ["news", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/news/_/id/2977187/cooper-kupp",
                          },
                          {
                            rel: ["bio", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/bio/_/id/2977187/cooper-kupp",
                          },
                          {
                            rel: ["overview", "desktop", "athlete"],
                            href:
                              "http://www.espn.com/nfl/player/_/id/2977187/cooper-kupp",
                          },
                        ],
                        headshot:
                          "https://a.espncdn.com/i/headshots/nfl/players/full/2977187.png",
                        jersey: "10",
                        position: { abbreviation: "WR" },
                        team: { id: "14" },
                        active: true,
                      },
                      team: { id: "14" },
                    },
                  ],
                },
              ],
            },
          ],
          notes: [],
          status: {
            clock: 900,
            displayClock: "15:00",
            period: 1,
            type: {
              id: "1",
              name: "STATUS_SCHEDULED",
              state: "pre",
              completed: false,
              description: "Scheduled",
              detail: "Mon, November 23rd at 8:15 PM EST",
              shortDetail: "11/23 - 8:15 PM EST",
            },
          },
          broadcasts: [{ market: "national", names: ["ESPN"] }],
          leaders: [
            {
              name: "passingYards",
              displayName: "Passing Leader",
              shortDisplayName: "PASS",
              abbreviation: "PYDS",
              leaders: [
                {
                  displayValue: "254-385, 2739 YDS, 23 TD, 7 INT",
                  value: 2739,
                  athlete: {
                    id: "2330",
                    fullName: "Tom Brady",
                    displayName: "Tom Brady",
                    shortName: "T. Brady",
                    links: [
                      {
                        rel: ["playercard", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/_/id/2330/tom-brady",
                      },
                      {
                        rel: ["stats", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/stats/_/id/2330/tom-brady",
                      },
                      {
                        rel: ["splits", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/splits/_/id/2330/tom-brady",
                      },
                      {
                        rel: ["gamelog", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/gamelog/_/id/2330/tom-brady",
                      },
                      {
                        rel: ["news", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/news/_/id/2330/tom-brady",
                      },
                      {
                        rel: ["bio", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/bio/_/id/2330/tom-brady",
                      },
                      {
                        rel: ["overview", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/_/id/2330/tom-brady",
                      },
                    ],
                    headshot:
                      "https://a.espncdn.com/i/headshots/nfl/players/full/2330.png",
                    jersey: "12",
                    position: { abbreviation: "QB" },
                    team: { id: "27" },
                    active: true,
                  },
                  team: { id: "27" },
                },
              ],
            },
            {
              name: "rushingYards",
              displayName: "Rushing Leader",
              shortDisplayName: "RUSH",
              abbreviation: "RYDS",
              leaders: [
                {
                  displayValue: "143 CAR, 730 YDS, 5 TD",
                  value: 730,
                  athlete: {
                    id: "3912550",
                    fullName: "Ronald Jones II",
                    displayName: "Ronald Jones II",
                    shortName: "R. Jones II",
                    links: [
                      {
                        rel: ["playercard", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/_/id/3912550/ronald-jones-ii",
                      },
                      {
                        rel: ["stats", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/stats/_/id/3912550/ronald-jones-ii",
                      },
                      {
                        rel: ["splits", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/splits/_/id/3912550/ronald-jones-ii",
                      },
                      {
                        rel: ["gamelog", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/gamelog/_/id/3912550/ronald-jones-ii",
                      },
                      {
                        rel: ["news", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/news/_/id/3912550/ronald-jones-ii",
                      },
                      {
                        rel: ["bio", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/bio/_/id/3912550/ronald-jones-ii",
                      },
                      {
                        rel: ["overview", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/_/id/3912550/ronald-jones-ii",
                      },
                    ],
                    headshot:
                      "https://a.espncdn.com/i/headshots/nfl/players/full/3912550.png",
                    jersey: "27",
                    position: { abbreviation: "RB" },
                    team: { id: "27" },
                    active: true,
                  },
                  team: { id: "27" },
                },
              ],
            },
            {
              name: "receivingYards",
              displayName: "Receiving Leader",
              shortDisplayName: "REC",
              abbreviation: "RECYDS",
              leaders: [
                {
                  displayValue: "53 REC, 577 YDS, 2 TD",
                  value: 577,
                  athlete: {
                    id: "2977187",
                    fullName: "Cooper Kupp",
                    displayName: "Cooper Kupp",
                    shortName: "C. Kupp",
                    links: [
                      {
                        rel: ["playercard", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/_/id/2977187/cooper-kupp",
                      },
                      {
                        rel: ["stats", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/stats/_/id/2977187/cooper-kupp",
                      },
                      {
                        rel: ["splits", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/splits/_/id/2977187/cooper-kupp",
                      },
                      {
                        rel: ["gamelog", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/gamelog/_/id/2977187/cooper-kupp",
                      },
                      {
                        rel: ["news", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/news/_/id/2977187/cooper-kupp",
                      },
                      {
                        rel: ["bio", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/bio/_/id/2977187/cooper-kupp",
                      },
                      {
                        rel: ["overview", "desktop", "athlete"],
                        href:
                          "http://www.espn.com/nfl/player/_/id/2977187/cooper-kupp",
                      },
                    ],
                    headshot:
                      "https://a.espncdn.com/i/headshots/nfl/players/full/2977187.png",
                    jersey: "10",
                    position: { abbreviation: "WR" },
                    team: { id: "14" },
                    active: true,
                  },
                  team: { id: "14" },
                },
              ],
            },
          ],
          tickets: [
            {
              summary: "Tickets as low as $134",
              numberAvailable: 93,
              links: [
                {
                  href:
                    "https://www.vividseats.com/nfl/tampa-bay-buccaneers-tickets/buccaneers-11-23-3525252.html?wsUser=717",
                },
                {
                  href:
                    "https://www.vividseats.com/venues/raymond-james-stadium-tickets.html?wsUser=717",
                },
              ],
            },
          ],
          startDate: "2020-11-24T01:15Z",
          geoBroadcasts: [
            {
              type: { id: "1", shortName: "TV" },
              market: { id: "1", type: "National" },
              media: { shortName: "ESPN" },
              lang: "en",
              region: "us",
            },
          ],
          odds: [
            {
              provider: { id: "38", name: "Caesars", priority: 1 },
              details: "TB -4.5",
              overUnder: 48,
            },
          ],
        },
      ],
      links: [
        {
          language: "en-US",
          rel: ["summary", "desktop", "event"],
          href: "http://www.espn.com/nfl/game/_/gameId/401220335",
          text: "Gamecast",
          shortText: "Gamecast",
          isExternal: false,
          isPremium: false,
        },
      ],
      weather: {
        displayValue: "Mostly cloudy",
        temperature: 71,
        conditionId: "38",
      },
      status: {
        clock: 900,
        displayClock: "15:00",
        period: 1,
        type: {
          id: "1",
          name: "STATUS_SCHEDULED",
          state: "pre",
          completed: false,
          description: "Scheduled",
          detail: "Mon, November 23rd at 8:15 PM EST",
          shortDetail: "11/23 - 8:15 PM EST",
        },
      },
    },
  });
  expect(wrapper.is("#gameInfoContainer")).toBe(true);
});
