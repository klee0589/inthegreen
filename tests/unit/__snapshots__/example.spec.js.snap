// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameInfo has the expected html structure 1`] = `<!---->`;
